operators/src/processing/interpolation/mod.rs

```diff
@@ -30,10 +30,17 @@ use snafu::{ensure, Snafu};
 #[serde(rename_all = "camelCase")]
 pub struct InterpolationParams {
     pub interpolation: InterpolationMethod,
-    pub input_resolution: SpatialResolution,
+    pub input_resolution: InputResolution,
 }
 
-#[derive(Debug, Serialize, Deserialize, Clone)]
+#[derive(Debug, Serialize, Deserialize, Clone, Copy)]
+#[serde(rename_all = "camelCase", tag = "type")]
+pub enum InputResolution {
+    Value(SpatialResolution),
+    Native,
```

Comment:
I mean the finest resolution that the operator can produce. Query it with any finer resolution and you get no more data but only interpolated data. Thus I called it `native` because it is the native resolution of the data. We can also call it `max` or `original`.

I don't think `derived` quite captures that, because you could also derive something else than the maximum reasonable resolution.

Reply:
cf. Slack

> Christian
  [18:27 Uhr](https://agdbs.slack.com/archives/C02JDR82DAP/p1663000043475689)
[@Michael M.](https://agdbs.slack.com/team/UBBE024JG)
 im UI nennst du das ja auch "Source Resolution". Vielleicht ist das besser als Native, denn Native ist ja komisch, wenn man eine setzt nach der Expression.


> Michael M.
  [18:32 Uhr](https://agdbs.slack.com/archives/C02JDR82DAP/p1663000372527469)
jo


> Christian
  [18:35 Uhr](https://agdbs.slack.com/archives/C02JDR82DAP/p1663000537185969)
Dann würde ich dir das noch mal assignen und danach kann man das erst einmal mergen.